open-source-terms.md

```diff
@@ -32,10 +37,10 @@ called _npm Services_ throughout these Terms.
 You may only access or use npm Services by agreeing to these Terms. If
 npm adds any additional functionality to npm Services, you must agree
 to these Terms to use new features, too. You show your agreement with
-npm on these Terms by clicking a check box in the process of creating an
-user account on the Website (your _Account_) or by accessing or using
-npm Services without creating an account. The agreement between you and
-npm is a legally binding contract (this _Agreement_).
+npm on these Terms by by creating a user account on the Website (your
```

Comment:
Good catch; it could be read as limiting the terms to accounts created via website. I had in mind that every account appears on the website, regardless of how you create it.